dashboard/app/models/script.rb

```diff
@@ -628,4 +644,14 @@ def self.clear_cache
   def localized_title
     I18n.t "data.script.name.#{name}.title"
   end
+
+  def self.build_property_hash(script_data)
+    Hash.new.tap do |property_hash|
+      property_hash[:pd] = script_data[:pd] || false # default false
+      property_hash[:admin_required] = script_data[:admin_required] || false # default false
+      property_hash[:professional_learning_course] = script_data[:professional_learning_course] || false # default false
+      property_hash[:student_of_admin_required] = script_data[:student_of_admin_required] || false # default false
+      property_hash[:peer_reviews_to_complete] = script_data[:peer_reviews_to_complete] if script_data[:peer_reviews_to_complete]
+    end
+  end
```

Comment:
So here, I'm using tap because I don't want to set the peer_reviews_to_complete key for scripts that don't have peer reviews.